app-misc/nnn/nnn-2.8.1.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="The missing terminal file manager for X"
+DESCRIPTION="nnn is a full-featured terminal file manager. It's tiny and \
+nearly 0-config with an incredible performance.
+
+nnn is also a du analyzer, an app launcher, a batch renamer and a file picker. \
+The plugin repository has tons of plugins and documentation to extend the \
+capabilities further. You can plug new functionality and play with a \
+custom keybind instantly. There's an independent (neo)vim plugin.
+
+It runs smoothly on the Raspberry Pi, Termux on Android, Linux, macOS, BSD, \
+Cygwin, WSL and works seamlessly with DEs and GUI utilities.
+
+Visit the Wiki for concepts, program usage, how-tos and troubleshooting."
+HOMEPAGE="https://github.com/jarun/nnn"
+COPYRIGHT="2016-2019 Arun Prakash Jana"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/jarun/nnn/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="cadf9cf8038433aeeb50a777180ad4b309ac7d2fec81c7da177ddca515812f06"
+PATCHES="nnn-$portVersion.patchset"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	nnn$secondaryArchSuffix = $portVersion
+	cmd:nnn = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libncurses$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:install
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make PREFIX=$prefix MANPREFIX=$manDir install
```

Comment:
Just checked out a srcGitRev from master there with your changes, added a comment on your merged PR @annagrram , the suggested change will remove the need to set MANPREFIX here :)

Reply:
PS, isn't there a way to rename "misc/haiku/Makefile" to "Makefile.haiku" in their github ? (maybe overkill) :)